float64 uav_num

float64 d1_x
float64 d1_y
float64 d1_z

float64 d2_x
float64 d2_y
float64 d2_z

float64 d3_x
float64 d3_y
float64 d3_z

float64 d1_change
float64 d2_change
float64 d3_change
